assets/src/dashboard/app/views/shared/bodyViewOptions.js

```diff
@@ -43,6 +43,7 @@ const DisplayFormatContainer = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
+  margin-top: -10px;
```

Comment:
`DisplayFormatContainer` gets its height from the theme (`formatContainer.height`). To give this area more space I increased that from 44px to 76px. Because of how the header space is allocated currently, the least invasive way to make this so that there was more space below the contents of this container than above while still centering the children was to give margin top a negative 10 value. When we make design updates this header area will change and allow us to use height differently.